Week-10/url-shortener/src/components/Header/style.css

```diff
@@ -0,0 +1,33 @@
+.headerContainer {
+  display: flex;
+  height: 80px;
+  width: 100%;
+  justify-content: space-between;
+  /* padg: 15px; */
+  align-items: center;
+  background-color: #232323;
```

Comment:
create variable file and reuse it across instead of writing same style in multiple css files